ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -143,7 +143,7 @@ public void run() {
 
         // Compaction doesn't work under a transaction and hence pass 0 for current txn Id
         // The response will have one entry per table and hence we get only one OpenWriteIds
-        String fullTableName = TxnUtils.getFullTableName(t.getDbName(), t.getTableName());
+        String fullTableName = Warehouse.getQualifiedName(t.getDbName(), t.getTableName());
```

Comment:
This uses default cat name - what is to ensure that compaction_queue.catalog is also default?

Reply:
Yes, I should fix this.